<?xml version="1.0"?>
<launch>
  <node pkg="rowbot_control" type="motor_controller.py" name="motor_controller">
    <remap from="left_cmd" to="left_thrust_cmd"/>
    <remap from="right_cmd" to="right_thrust_cmd"/>
  </node>

  <node name="course_to_twist" pkg="rowbot_control" type="course_to_twist" output="screen">
    <param name="~course_kp" value="0.5"/>
  </node>
</launch>
